eliHenderson/assignments/fastSlow/lc876.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Summary: 
+ *  If there is no cycle in the linked list 
+ *  there will be a null at the of list
+ *  we can use two pointer to iterate through the 
+ *  array. One pointer will move 2x faster than the 
+ *  other, Once the fast pointer reaches the end, 
+ *  the slower pointer will be a the middle
+ *  return the slow pointer
+ * 
+ * while (fast is not null)
+ *  fast = fast.next.next
+ *  slow = slow.next
+ * 
+ * return slow
```

Comment:
I love this summary. However, for the psuedocode, I would recommend to write out in full sentences about what your code will be doing. That way, anyone who comes and checks out your code can see in a high-level what your function is trying to accomplish.